yaml-parser.js

```diff
@@ -20,27 +20,66 @@ function parseCollectionYaml(filePath) {
       let currentArray = null;
       let currentObject = null;
 
+      const readLiteralBlock = (startIndex, parentIndent) => {
```

Comment:
Unrelated to this PR,

Feel free to ignore: @aaronpowell, any reason we didn't want to use a yaml parsing npm package, https://www.npmjs.com/package/yaml

Would this added complexity justify bringing in an npm package?